libr/debug/p/native/w32.c

```diff
@@ -245,69 +243,68 @@ static int w32_dbg_init() {
 	w32dbg_SeDebugPrivilege ();
 	/* lookup function pointers for portability */
 	w32_DebugActiveProcessStop = (BOOL (WINAPI *)(DWORD))
-		GetProcAddress (GetModuleHandleA ("kernel32"),
+		GetProcAddress (GetModuleHandle (TEXT ("kernel32")),
```

Comment:
Use `GetModuleHandleW(L"kernel32")` here explicitly because UNICODE might not be defined and you always want the wide variant anyway...

Reply:
Also, just cache this because the module handle for kernel32 is being used here quite a lot!

Reply:
UNICODE is already defined in file mk/platform.mk
You're right, not cached, w32 code need refactory a lot.